helm/istio/charts/sidecarInjectorWebhook/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
I wonder if this change should be moved into maistra/istio?  Upstream istio provides this file and the `.tpl` file was added by maistra.  In other words, should we revert part of these changes, maistra/istio#6?

Reply:
I asked @knrc about rolling back those changes and he said he was assigned an upstream issue to implement them upstream, so no. See comments here: https://github.com/Maistra/istio/pull/36